Sources/SwiftCliCore/ChartConstructor.swift

```diff
@@ -81,8 +51,35 @@ public class ChartConstructor {
             rightStitches = "─┼─┐\n"
         case (rDiff == 0):
             rightStitches = "─┤\n"
+        case (rDiff == 1):
+            rightStitches = "─┼─┘\n"
         default:
-            rightStitches = "─┤\n"
+            let midRightSt = String(repeating: "─┴", count: ((abs(rDiff))-1))
+            rightStitches = "─┼\(midRightSt)─┘\n"
+        }
+
+        switch true {
+        case (lDiff == 0):
+            leftStitches = "├"
+        case (lDiff == 1):
+            leftStitches = "└─┼"
+        case (lDiff >  1):
+            let midLeftSt = String(repeating: "─┴", count: ((abs(lDiff))-1))
+            leftStitches = "└\(midLeftSt)─┼"
+        default:
+            leftStitches = "├"
```

Comment:
I would fold the `lDiff == 0` case into the `default` if there's nothing different between the two.